Adding section for monitoring with PKI authentication (#14227)
elastic/beats: libbeat/docs/monitoring/monitoring-internal-collection.asciidoc

call the `GET /` API against that cluster.
<2> This setting identifies the hosts and port numbers of {es} nodes
that are part of the monitoring cluster.
+
If you want to use PKI authentication to send monitoring events to
{es}, you must specify a different set of configuration options. For
example:
+

> ["source","yml",subs="attributes"]

Reviewer: ```suggestion
[source,yaml]
```

Reviewer: No need to complicate this with subs=attributes since the example doesn't contain any attributes that need to be resolved by asciidoctor.

--------------------
monitoring:
  enabled: true

>   cluster_uuid: PRODUCTION_ES_CLUSTER_UUID <1>

Reviewer: ```suggestion
  cluster_uuid: PRODUCTION_ES_CLUSTER_UUID
```

Reviewer: I find the callouts to be kind of confusing actually (and they don't render well in the HTML when you have multiple callouts with the same number), so I suggest removing them.

  elasticsearch:

>     hosts: ["https://example.com:9200", "https://example2.com:9200"] <2>

Reviewer: ```suggestion
    hosts: ["https://example.com:9200", "https://example2.com:9200"]
```

>     username: "" <1>

Reviewer: ```suggestion
    username: ""
```

>     ssl: <2>

Reviewer: ```suggestion
    ssl:
```

      ssl.certificate_authorities: ["/etc/pki/root/ca.pem"]
      ssl.certificate: "/etc/pki/client/cert.pem"
      ssl.key: "/etc/pki/client/cert.key"
--------------------

> <1> This setting must be explicitly specified so that the username from

Reviewer: ```suggestion
+
You must specify the `cluster_uuid` and `username` explicitly so that
the username from the client certificate (`CN`) is used. See
<<configuration-ssl>> for more information about SSL settings.
```

> the client certificate (`CN`) is used.

Reviewer: ```suggestion
```

> <2> See See <<configuration-ssl>> for more information.

Reviewer: ```suggestion
```

call the `GET /` API against that cluster.
<2> This setting identifies the hosts and port numbers of {es} nodes
that are part of the monitoring cluster.
+
If you want to use PKI authentication to send monitoring events to
{es}, you must specify a different set of configuration options. For
example:
+
[source,yaml]
--------------------
monitoring:
  enabled: true
  cluster_uuid: PRODUCTION_ES_CLUSTER_UUID
  elasticsearch:
    hosts: ["https://example.com:9200", "https://example2.com:9200"]
    username: ""
    ssl:
      ssl.certificate_authorities: ["/etc/pki/root/ca.pem"]
      ssl.certificate: "/etc/pki/client/cert.pem"
      ssl.key: "/etc/pki/client/cert.key"
--------------------
+

> You must specify the `cluster_uuid` and `username` explicitly so that

Author: Just `username` here, not `cluster_uuid`. I will fix.

Author: Fixed in 4c61ebc.
